compiler/src/iree/compiler/Codegen/Common/test/specialize_exports.mlir

```diff
@@ -0,0 +1,378 @@
+// RUN: iree-opt %s \
+// RUN:   --pass-pipeline="builtin.module(hal.executable(hal.executable.variant(iree-codegen-specialize-exports, cse)))" \
+// RUN: --split-input-file | FileCheck %s
+
+#executable_target_embedded_elf_aarch64 = #hal.executable.target<"llvm-cpu", "embedded-elf-aarch64">
+#pipeline_layout = #hal.pipeline.layout<constants = 1, bindings = [
+  #hal.pipeline.binding<storage_buffer, "ReadOnly|Indirect">,
+  #hal.pipeline.binding<storage_buffer, ReadOnly>,
+  #hal.pipeline.binding<storage_buffer, Indirect>], flags = Indirect>
+module {
```

Comment:
```suggestion
```

remove top-level modules - they aren't needed and will let you unindent all the lines by 2 spaces
(here and below)